### ISSUE TITLE
[New] pls add auto-update-download

### ISSUE THREAD

**Issue by CkaDebug** [Dreeam-qwq/Leaf#27]
求添加一个自动下载leaf的最新构建的功能

**Comment by Dreeam-qwq** [comment]
以后会加，但是优先度很低

**Comment by TheFloodDragon** [comment]
如果我理解的没错话的，你不是可以在GithubActions里下载到最新的自动构建版本吗，或者在Releases里

**Comment by HaHaWTH** [comment]
> 如果我理解的没错话的，你不是可以在GithubActions里下载到最新的自动构建版本吗，或者在Releases里

Yep, but automatic update is more convenient, Leaves has already done so
